src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -122,6 +122,7 @@
         </service>
 
         <service id="translation.loader" class="Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader" public="true" />
+        <service id="translation.reader" class="Symfony\Component\Translation\Reader\TranslationReader" public="true" />
```

Comment:
the new service should be private, but we have to merge https://github.com/symfony/symfony/pull/23624 first or make this one private once https://github.com/symfony/symfony/pull/23624 merged because the `debug:translation` command `get`s it from the container. Let's keep it as is for now